cves/kernel/CVE-2020-16166.yml

```diff
@@ -239,10 +161,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    There is no evidence as to how this bug was discovered, however, we
+    do know who reported it, Amit Klein. He also later released statements
```

Comment:
Where did you find out who reported it? I think that piece of evidence should be backed up a little more.

Reply:
Information was found in commit messages as well as being on almost every page discussing the vulnerability. That information will be added in the next commit.